include/common-types.h

```diff
@@ -0,0 +1,25 @@
+#ifndef CONNECT_SDK_COMMON_TYPES_H
+#define CONNECT_SDK_COMMON_TYPES_H
+
+#include <string>
+#include <vector>
+#include <map>
+#include <boost/move/unique_ptr.hpp>
+#include <boost/chrono/system_clocks.hpp>
+
+namespace prism {
+namespace connect {
+
+using std::string;
```

Comment:
Why we define this in header file? What happen when somebody will include it in another project and write

using namespace prism::connect;
using namespace std;

That using stuff should be used only in *.cpp files.